tests/saw/spec/handshake/rfc-handshake.cry

```diff
@@ -132,6 +132,7 @@ connectionParameters conn params =
  /\ (~params.includeSessionTicket) // s2n server does not issue tickets at this time
  /\ (~params.renewSessionTicket) // s2n server does not issue tickets at this time
  /\ conn.client_auth_flag == params.requestClientCert
+ /\ conn.actual_protocol_version != S2N_TLS13
```

Comment:
Given this RFC  is setup specifically for TLS1.2 maybe check for protocol is `== S2N_TLS12` instead of `!=S2N_TLS13` ?

Reply:
This RFC is actually specifically set up for anything before 1.3 :) They all use the same state machine, and actual version shouldn't have any effect. We continue proving that if we use !=S2N_TLS13, which just ignores the specific case that we know won't work.